packages/lesswrong/components/form-components/SequencesListEditorItem.tsx

```diff
@@ -57,12 +57,6 @@ const SequencesListEditorItem = ({documentId, classes, ...props}) => {
           <div className="sequences-list-edit-item-author">
             {(document.user && document.user.displayName) || "Undefined Author"}
           </div>
```

Comment:
Was this part intentional? Seems kinda random?

Reply:
Yes. I added type annotations to `useSingle`, because it would've caught the moderation guidelines crash, and this meant having to clean up this bit because it didn't type-check.